### PR TITLE
Bootstrap CoC, Coach Guide, Student Guide and FAQ pages

### PR DESCRIPTION
**Pull request by KimberleyCook**
| Code of Conduct | Coach Guide |
|-------|------|
|screencapture-localhost-3000-code-of-conduct-2020-11-26-17_18_53|screencapture-localhost-3000-effective-teacher-guide-2020-11-27-14_34_45|

| Student Guide | FAQ |
|-------|------|
|screencapture-localhost-3000-student-guide-2020-11-27-14_35_35|screencapture-localhost-3000-faq-2020-11-26-17_28_12|

### CODE REVIEW

File: app/views/dashboard/code.html.haml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-.stripe.reverse#code-of-conduct
```
Comment (matyikriszta) [inline app/views/dashboard/code.html.haml:1]: @KimberleyCook I think this id was used as an anchor tag so can we add it back?

**Comment by matyikriszta** [comment]
@KimberleyCook this looks good, I have one general comment. I really dislike the mismatched font weight between paragraph and list items, do you think we could update this so they are the same weight? This has been bugging me for a while so it would be good to address it as part of this PR.

File: app/views/dashboard/code.html.haml (hunk 1 above)
Comment (KimberleyCook) [reply app/views/dashboard/code.html.haml:1]: Ah ok cool, I tried looking but couldn't find where it was used. I could see the ones on the eligibility page.

**Comment by KimberleyCook** [comment]
@matyikriszta I've put p tags inside the li's making the styles the same. I've also updated the screenshots.

**Review by matyikriszta — APPROVED** [review]
👍

**Pull request merged**